templates/jobs/main.html

```diff
@@ -40,8 +38,12 @@
                         Jobs
                     </div>
                     <div class="panel-body">
-                        <p>Here you can find job offers for an entry-level
-                            positions.</p>
+                        <p>
+                            Working as a software developer is a great way to become one. There is lots of companies out there who are hiring, but how to tell which one is good?
```

Comment:
Working as a software developer is a great way to become one. There are lots of companies out there who are hiring, but how can you tell which ones are good?